New version

melcidar opened the pull request:
SENDING EMAILS 

- using Nodemailer
- testing it by using Postman

melcidar committed:
New version

melcidar committed:
SENDING EMAILS

--------------------------------------------------------------------
 MAIN
 * added error message in configuration/error.js
 * used res.json to send status in controllers/sendemail.j
 * moved process.env to configuration/env.js
 * fixed some errors in controllers/sendemail.js
--------------------------------------------------------------------
 OTHER
 * those are changes since the last review
--------------------------------------------------------------------

becarselman commented on controllers/sendemail.js:
```diff
@@ -0,0 +1,13 @@
+
+    try {
+        await sendMailService.sendEmail(email, templateName)
+        res.status(200).send("OK")
```
Change this to json instead of send and send {message: ' some success message'}

becarselman reviewed (commented):
I hope you are test this and everything works well. Please add pr description.

melcidar committed:
TESTING SENDING EMAILS

--------------------------------------------------------------------
 MAIN
 * use res.json to send status and message
 * fixed few errors in services/sendemail.js
--------------------------------------------------------------------
 OTHER
 * tested sending mails using Postman, all mails were sent from
 project.pharaoh@hotmail.com. I tested sending mails to domains:
 gmail, hotmail, outlook, yahoo and aol. Everything was okay.
--------------------------------------------------------------------

melcidar committed:
SENDING MAILS - FINAL VERSION

--------------------------------------------------------------------
 MAIN
 * added function for extracting subject from the template using fs module
 * tested using Postman
--------------------------------------------------------------------
 OTHER
 * //// ////
--------------------------------------------------------------------

TarikVelic reviewed (approved):
good job, lgtm

melcidar committed:
Merge branch 'develop' into FEATURE/Meliha01/sendEmails_clean
